src/main/java/pl/put/poznan/builder/logic/Attribute.java

```diff
@@ -26,6 +26,9 @@ public Attribute addValue(String value) {
         this.values.addElement(value);
         return this;
     }
+    public Vector<String> getValue(){
```

Comment:
Plz leave one empty line between the end of one method and the declaration of the next one, here and in the rest of this pull request. There should also be a space before the opening curly brace. It's recommended by many styleguides like here: https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace. The code looks prettier with these rules 🙂